pkg/cmd/config/command.go

```diff
@@ -92,26 +110,19 @@ func NewCommand(ctx context.Context) *cli.Command {
 	return cmd
 }
 
-func Generate(values map[string]interface{}, outputFile string) error { //nolint: gofmt
-	t := template.New("template")
-	t, err := t.Parse(templateString)
+func Generate(data ScrapeConfigData) (string, error) {
+	t, err := template.New("scrape_config").Parse(scrapeConfigTemplate)
 	if err != nil {
-		return errors.Wrap(err, "template parser")
+		return "", errors.Wrap(err, "template parser")
 	}
-	var cleanup = func() {}
-	out := os.Stdout
-	if outputFile != "" {
-		output, err := os.Create(outputFile)
-		if err != nil {
-			return errors.Wrap(err, "creating output file")
-		}
-		cleanup = func() {
-			output.Close()
-		}
-		out = output
+
+	var result strings.Builder
```

Comment:
See suggestion for template:
1. Remote Write Endpoint (with region, cloudAccountID, clusterName)